tools/get_release_contributors.py

```diff
@@ -237,21 +249,25 @@ def find_contributors(git_log_selector, from_, to):
         if contributor:
             contributors.add(contributor)
         else:
-            contributors.add(realname)
+            # TODO: check this is the right logic...
+            contributors.add(co_author)
     return sorted(contributors, key=str.casefold)
```

Comment:
Actually, you can probably try this script yourself as you're a new contributor. 
Yes, Co-authoring might be a bit tricky.

Reply:
Good idea! I'll do that now:

```
$ tools/get_release_contributors.py --from 5ff7e633 --to $(git rev-parse HEAD) --repo .
WARNING unable to find a github profile with public email nick@REDACTED (me)
WARNING unable to find a github profile with public email peter@REDACTED
WARNING unable to find a github profile with public email actions@github.com
Code contributors: @ann0see, @declension, @pljones
Application translators: @ann0see
```
is this good?

Reply:
Application translator @ann0see looks a bit strange

Reply:
:information_source:  That's also the case on `main`:

```
$ git checkout main && tools/get_release_contributors.py --from 5ff7e633 --to $(git rev-parse HEAD) --repo .
Code contributors: @ann0see, @pljones
Application translators: @ann0see
```